ooni/nettest.py

```diff
@@ -591,11 +594,16 @@ def generateMeasurements(self):
                 test_instance._setUp()
                 test_instance.summary = self.summary
                 for method in test_methods:
+                    try:
+                        measurement = self.makeMeasurement(
```

Comment:
So thinking a bit more about this, what is not nice is that the constructor for `Measurement` does a bunch of things, including calling `setUp` that is a method that can actually throw an exception (https://github.com/TheTorProject/ooni-probe/blob/99367d6b724b69897215230fa58dc398eabc42db/ooni/tasks.py#L114).

I guess the cleaner way of managing this would be to move the setup of the NetTestCase instance inside of another method that is called inside of `makeMeasurement` and handle exceptions in there.